libvast/include/vast/defaults.hpp

```diff
@@ -219,7 +219,7 @@ inline constexpr size_t max_partition_size = 4'194'304; // 4 Mi
 
 /// Timeout after which an active partition is forcibly flushed.
 inline constexpr caf::timespan active_partition_timeout
-  = std::chrono::minutes{5};
+  = std::chrono::seconds{30};
```

Comment:
These two also need to be adapted in the following places:
- application.cpp (help text)
- tenzir.yaml.example (default values)

And they likely also need a changelog entry.